extensions/pathfactory/pathfactory-content-track.schema.json

```diff
@@ -0,0 +1,236 @@
+{
```

Comment:
Most of the descriptions needs to have more detailed descriptions for users to understand what these fields are and what are their use case / usage